Source/SwiftLintBuiltInRules/Rules/Lint/PrivateSwiftUIStatePropertyRuleExamples.swift

```diff
@@ -0,0 +1,243 @@
+internal struct PrivateSwiftUIStatePropertyRuleExamples {
```

Comment:
I added a couple new test cases to cover existing behavior and some corrections to validate the new Rewriter, but that put the rule declaration file over the 400 line limit. So I went ahead and took the opportunity to move the examples into their own file. Let me know if there's a better option!